app/utils/statusHelper.py

```diff
@@ -0,0 +1,22 @@
+
+import Enum
+class Status(Enum):
+	UPLOADING = "Uploading video"
+	TRACKING_FEATURES = "Tracking features"
+	TRACKING_OBJEcTS = "Tracking objects"
+	GENERATING_VIDEOS = "Generating short videos"
+	COMBINING_VIDEOS = "Combining videos"
+	SAFETY_ANALYSIS = "Running safety analysis"
+	DONE = "Done"
+
+class StatusHelper(object):
```

Comment:
Where does `StatusHelper` get used?   If this PR is a WIP, then I expect these classes to be used in the StatusHandler post/get methods.  Am I right about this guess?